pom.xml

```diff
@@ -600,6 +604,110 @@
 				<opensrp.database.profile>postgres</opensrp.database.profile>
 			</properties>
 		</profile>
+		<profile>
+			<id>FileSystem</id>
+			<activation>
+				<activeByDefault>true</activeByDefault>
+				<property>
+					<name>opensrp.file.manager.type</name>
+					<value>FileSystem</value>
+				</property>
+			</activation>
+			<properties>
+				<opensrp.file.manager.profile>FileSystem</opensrp.file.manager.profile>
+			</properties>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.apache.maven.plugins</groupId>
+						<artifactId>maven-compiler-plugin</artifactId>
+						<configuration>
+							<excludes>
+								<exclude>**/OSSMultimediaFileManager.*</exclude>
+								<exclude>**/OSSClientBuilder.*</exclude>
+								<exclude>**/S3MultimediaFileManager.*</exclude>
+							</excludes>
+<!--							<testExcludes>-->
```

Comment:
I have temporarily commented out these files. Will clean code once you provide your feedback. Thanks @githengi